components/eam/src/physics/cam/nucleate_ice_cam.F90

```diff
@@ -474,15 +469,12 @@ subroutine nucleate_ice_cam_calc( &
    do k = top_lev, pver
       do i = 1, ncol
 
-         if (t(i,k) < tmelt - 5._r8) then
+         if (temperature(i,k) < tmelt - 5._r8) then
```

Comment:
I like the name `temperature`, but I see the temperature referred to as `t` elsewhere in this PR (and in others). Do we want to attempt to make this and other variable names consistent within the code base?

Reply:
replaced t with temperature in both nucleate_ice_cam_calc and microp_aero_run